schema.sql

```diff
@@ -25,4 +25,28 @@ CREATE TABLE species (
     id INT GENERATED ALWAYS AS IDENTITY,
     name TEXT
 );
-ALTER TABLE species ADD PRIMARY KEY (id);
\ No newline at end of file
+ALTER TABLE species ADD PRIMARY KEY (id);
+
+CREATE TABLE vets (
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY, 
+    name TEXT, 
+    age INT, 
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    species_id INT, 
+    vet_id INT,
+);
```

Comment:
- [ ] Please remove comma at the end of line 39 because it is causing an syntax error :thinking: